modules/nix/nixpkgs.nix

```diff
@@ -44,72 +90,258 @@ let
     check = lib.isFunction;
     merge = lib.mergeOneOption;
   };
+
+  pkgsType = types.pkgs // {
```

Comment:
`types.pkgs` was added in 23.05. What’s the minimum nixpkgs supported by nix-darwin?

Reply:
We don't really have any, but dropping 22.11 support immediately in the docs PR was considered too soon so I've pushed another backporting workaround. (We really need release branches.)